src/sp/it/pl/util/system/Environment.kt

```diff
@@ -49,8 +49,8 @@ fun copyToSysClipboard(df: DataFormat, o: Any?) = o?.let { Clipboard.getSystemCl
 
 /**
  * Launches this file as an executable program as a separate process. Does not wait for the program or block.
- * * working directory of the program will be set to the parent directory of its file
- * * the program may start as a child process if otherwise not possible
+ * - working directory of the program will be set to the parent directory of its file
```

Comment:
Can you explain the reasoning behind the change of * vs -

Reply:
only a visual thing - since the lines start with *, the - is better distinguishable